misc/git.talon

```diff
@@ -1,4 +1,5 @@
 tag: terminal
+tag: git
```

Comment:
Where is this tag defined?

Reply:
To activate a tag, you do
```
-
tag(): terminal
```

though I'm not sure we'd want to bulk enable terminal commands in VSCode, might cause issues.

Note that custom tags need to be declared in .py, e.g.
https://github.com/knausj85/knausj_talon/blob/41760365bafcb9672e0c570de774d8aba4192e16/code/gdb.py#L5